src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: block;
+    width: 16px;
+    height: 16px;
+    background: url("../images/star.svg") no-repeat center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--1 &__star:nth-child(-n + 1),
+  &--2 &__star:nth-child(-n + 2),
+  &--3 &__star:nth-child(-n + 3),
+  &--4 &__star:nth-child(-n + 4),
+  &--5 &__star:nth-child(-n + 5) {
+    background-image: url("../images/star-active.svg");
+  }
```

Comment:
<img width="622" alt="Screenshot 2023-08-23 at 15 17 48" src="https://github.com/mate-academy/layout_catalog/assets/135232001/eb8f5c97-e72c-47f1-8eab-d5cdff67b984">
Consider following the checklist for this task